app/src/main/java/org/fossasia/susi/ai/chat/ChatActivity.kt

```diff
@@ -101,6 +106,25 @@ class ChatActivity : AppCompatActivity(), IChatView {
             }
         }
     }
+    override fun onTouchEvent(event: MotionEvent): Boolean {
+        this.gestureDetectorCompat?.onTouchEvent(event)
+        return super.onTouchEvent(event)
+    }
+    internal inner class CustomGestureListener : GestureDetector.SimpleOnGestureListener() {
+
+        override fun onFling(
+            event1: MotionEvent,
+            event2: MotionEvent,
+            velocityX: Float,
+            velocityY: Float
+        ): Boolean {
+            if (event2.x < event1.x) {
```

Comment:
This will require very small gesture

Reply:
Sir, shall I use manual values like 
<img width="660" alt="Screenshot 2019-03-30 at 2 01 43 AM" src="https://user-images.githubusercontent.com/43731599/55261000-d347a680-528f-11e9-8440-c287dafb31cf.png">
this to handle gesture.

Reply:
Yes, I'm not an expert on gesture but a nudge should not open the activity